### PR TITLE
Disable save without peer data and handle errors of invalid tokens

### PR DESCRIPTION
**Pull request by penk85**
- WPC-262: Set the peered data to display & button to save peer only after a token has been entered.
- WPC-263: Setup error handling on token so an unauthenticated token won't log the user out. Further improvements on token validation with WPC-298.

### CODE REVIEW

File: packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap [hunk 1]
```diff
@@ -552,6 +552,19 @@ exports[`Peer Edit Panel should be able to generate a new peering 1`] = `
...
+                              <defaultProps(HelpBlock)
+                                classPrefix="rs-help-block"
+                              >
+                                <HelpBlock
+                                  classPrefix="rs-help-block"
+                                >
+                                  <span
+                                    className="rs-help-block"
+                                  >
+                                    peerList.panels.validateToken
```
Comment (bishoymelek) [inline packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap:564]: Do we need here to use `t function` ?

File: packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap [hunk 2]
```diff
@@ -1233,6 +1246,17 @@ exports[`Peer Edit Panel should render 1`] = `
                                   </FormControl>
                                 </FormControlWrapper>
                               </defaultProps(FormControlWrapper)>
+                              <defaultProps(HelpBlock)
+                                classPrefix="rs-help-block"
+                              >
+                                <HelpBlock
+                                  classPrefix="rs-help-block"
+                                >
+                                  <span
```
Comment (bishoymelek) [inline packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap:1255]: should the help block has value?

File: packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap [hunk 3]
```diff
@@ -1943,6 +1967,17 @@ exports[`Peer Edit Panel should render with ID 1`] = `
...
+                                    className="rs-help-block"
```
[the remark this answers is not in the document]
Comment (roosnic1) [reply packages/editor/__tests__/specs/__snapshots__/peerEditPanel.tsx.snap:1977]: @bishoymelek this is the test snapshot

**Review by roosnic1 — APPROVED** [review]
👍🏾

**Review by bishoymelek — APPROVED** [review]

**Review by HanaAyman — APPROVED** [review]

**Pull request merged**